src/Api.js

```
@@ -32,7 +32,13 @@ class Api {
     this.token = null;
     this.apiReference = apiServices.register(this);
     this.baseUrl = (() => {
-      if (typeof baseUrl === "string" && baseUrl.substr(-1) !== "/") {
+      if (!baseUrl) {
+        console.error(
+          "SPQ Redux Api Client: Api Services should have a baseUrl"
```

Perhaps we should just throw an error here and allow them to catch it (and have advantage of a traceback).